tests/test_kernel_execution.py

```diff
@@ -266,12 +266,15 @@ def simpleKernel(particle, fieldset, time):
 
 
 @pytest.mark.parametrize('delete_cfiles', [True, False])
-def test_execution_keep_cfiles(fieldset, delete_cfiles):
+def test_execution_keep_cfiles_and_nocompilation_warnings(fieldset, delete_cfiles):
     pset = ParticleSet(fieldset, pclass=JITParticle, lon=[0.], lat=[0.])
-    pset.execute(pset.Kernel(DoNothing, delete_cfiles=delete_cfiles), endtime=1., dt=1.)
+    pset.execute(pset.Kernel(AdvectionRK4, delete_cfiles=delete_cfiles), endtime=1., dt=1.)
```

Comment:
the test still fails, and I can imagine that is because `pset.Kernel(..., endtime=1., dt=1.)` - the main func actually does not progress to the next step. Does `pset.Kernel(..., endtime=1.1, dt=1.)` make a difference for the online CI OSX test ?